tests/embedders/hex2vec/generation.py

```diff
@@ -32,11 +33,10 @@ def generate_test_case(
     regions_indexes = neighbourhood.get_neighbours_up_to_distance(root_region_index, radius)
     regions_indexes.add(root_region_index)
     regions_indexes = list(regions_indexes)  # type: ignore
-    regionalizer = H3Regionalizer(h3_res)
 
-    geoms = [regionalizer._h3_index_to_shapely_polygon(r) for r in regions_indexes]
+    geoms = cells_to_polygons([h3.str_to_int(r) for r in regions_indexes]).values
```

Comment:
Why do you skip using our regionalizer here?

Reply:
Function `_h3_index_to_shapely_polygon` has been completely removed from its logic. We can create a utility function for this use case.

Reply:
Ok, since it is independent from regionalizer instance anyway we can keep it like that IMHO. Are we sure we do not use this function anywhere in example notebooks?

Reply:
No, it's not used anywhere in examples.